tests/lca/imagebasedinstall/mgmt/deploy/internal/networkconfig/networkconfig.go

```diff
@@ -0,0 +1,37 @@
+package networkconfig
+
+import (
+	"fmt"
+	"net"
+)
+
+// IncreaseIP takes a network address and returns an IP based on the provided amount to increase the network by.
```

Comment:
Worth rephrasing - not clear if the network is expanded or another IP is returned.

Reply:
What do you mean by "not clear if the network is expanded"? The comment says that it takes a network address and returns an IP based on the increase argument. I.e. a network 192.168.123.0/24 with increase 1, returns 192.168.123.1.

Reply:
Is it clearer now?